### DISCUSSION TITLE
Prefect - ModuleNotFoundError

### DISCUSSION THREAD

**Discussion by rileyhun** [PrefectHQ/prefect#3076] [Q&A]
Hi everyone, I'm relatively new to Prefect. I have some custom classes separated in different python files that I turned into Prefect tasks. How do I go about importing from these classes and then running my flow on a Dask Cluster without getting a "ModuleNotFoundError"? Most other users, I noticed, have their tasks defined in the same file as their flow. Do the tasks and the flow have to be in the same python file?

**Comment by joshmeek** [comment]
Hi @rileyhun when using dask, any packages / modules that your flow code uses will also need to be importable from that worker. Not sure what your set up is but if you are running the dask workers locally and in different processes you need to ensure that your modules are importable from your `PYTHONPATH` wherever those workers run.
